misc/vasp/python/vasputil.py

```diff
@@ -41,7 +41,7 @@ def parseFile(file):
         pos.append(list(map(float, line.strip().split())))
     pos = numpy.array(pos).astype(numpy.float32)
 
-    factor = scale if scale >= 0.0 else np.power(
+    factor = scale if scale >= 0.0 else numpy.power(
         -scale / np.dot(a3, np.cross(a2, a1)), 1/3)
```

Comment:
Please change "np" to "numpy" in this line as well.

Reply:
Yes of course, I have changed it now